app/javascript/controllers/edit_fmea_controller.js

```diff
@@ -2,18 +2,56 @@ import { Controller } from "stimulus"
 import { toggleClass } from "helpers/index"
 
 export default class extends Controller {
-  static targets = ["card"]
+  static targets = ["card", "failuremodeForm", "causeForm", "effectForm"]
+
+  initialize() {
+    const selectedItemType = this.data.get("selected-item-type")
+    const selectedItemId = parseInt(this.data.get("selected-item-id"))
+  }
 
   connect() {
-    // if statement to only display copy button if browser
-    // support the action
+    console.log("--> edit fmea controller connected")
    }
 
+   createFailureMode () {
+    // need to overwrite the id of the parent element in the modal form with the correct value
+    this.failuremodeFormTarget.value = this.selectedItemId;
+  }
+
+   createCause () {
+    // need to overwrite the id of the parent element in the modal form with the correct value
+    this.causeFormTarget.value = this.selectedItemId;
+  }
+
+   createEffect () {
+    // need to overwrite the id of the parent element in the modal form with the correct value
+    this.effectFormTarget.value = this.selectedItemId;
+  }
+
```

Comment:
- Added method for each action button
- Added getters and setters below. 
  - This mean than any method in the controller knows what is currently selected